interfaces/builtin/desktop.go

```diff
@@ -609,9 +617,51 @@ func (iface *desktopInterface) AppArmorPermanentSlot(spec *apparmor.Specificatio
 	return nil
 }
```

Comment:
it's probably a good idea to add specific tests about tthe "deny-*connection" rules here now that I have moved to the plug side

Reply:
There is a test for slot "allow/deny-installation" in baseDeclSuite.TestSlotInstallation
https://github.com/canonical/snapd/blob/af81a100792825a10430c461416896f4c3e70b3d/interfaces/policy/basedeclaration_test.go#L924-L935
There is also a test for plug-side "allow-installation" in baseDeclSuite.TestDesktopFileIDsOverride.

I will now add specific tests for deny* rule, thanks for pointing that out.

Reply:
Done